staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -145,12 +167,13 @@ func readExpBackoffConfig() BackoffManager {
 	backoffBaseInt, errBase := strconv.ParseInt(backoffBase, 10, 64)
 	backoffDurationInt, errDuration := strconv.ParseInt(backoffDuration, 10, 64)
 	if errBase != nil || errDuration != nil {
-		return &NoBackoff{}
+		return defaultBackoffManager()
```

Comment:
is this right? 
if there is an error reading the config I think that we should not use a backoff manager instead of using the default

Reply:
with this PR the default is 'exponential backoff' - so if there is an error this means that we can't honor the values they specified in the env variables, instead we are going to use the default values. But they are still getting exponential backoff as they originally asked for.